File: OpenCV.Algorithmic.Solution/chapter-03/3.3.1/example1.py
# -*- coding: utf-8 -*-
import math

# 以(11,13)及(3,5)为中心进行极坐标变换 
r = math.sqrt(math.pow(11-3,2)+math.pow(13-5,2))
theta = math.atan2(13-5,11-3)/math.pi*180   # 转换为角度
print((r, theta))

File: OpenCV.Algorithmic.Solution/chapter-03/3.3.1/example1.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main()
{
    // 以(11,13)及(3,5)为中心进行极坐标变换 
    double r = sqrt(pow(11.0-3.0, 2)+pow(13.0-5.0, 2));
    double theta = atan2(13-5, 11-3)/3.14159*180;
    cout << r << ", " << theta << endl;
}

